src/main/java/org/jabref/gui/filelist/LinkedFilesEditDialogViewModel.java

```diff
@@ -91,9 +91,7 @@ public void openBrowseDialog() {
 
     public void setValues(LinkedFile linkedFile) {
         description.set(linkedFile.getDescription());
-
-        Path linkPath = Paths.get(linkedFile.getLink());
-        link.set(relativize(linkPath));
+        link.setValue(linkedFile.getLink()); //Might be an URL
```

Comment:
Now the `relativize` command is missing, right?
Maybe have something like `if (linkedFile.isOnline()) link.setValue(...) else link.setValue(relativize(Paths.get(...)))`?

Reply:
The relativize command was creating an error, if the url is a string.
I also tested the rename functionality and it's fine, no need for the relativizere here I think

Reply:
The relativize is only for the display, and I would prefer to keep it.
I was proposing the if (linkedFile.isOnline()) check to see if the string is an url.